rest/route/project_copy.go

```diff
@@ -93,6 +94,10 @@ func (p *projectCopyHandler) Run(ctx context.Context) gimlet.Responder {
 	if err = p.sc.CopyProjectSubscriptions(oldId, projectToCopy.Id); err != nil {
 		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "error copying subscriptions from project '%s'", p.oldProject))
 	}
+	//update admins
+	if err = p.sc.UpdateAdminRoles(projectToCopy, adminsToAdd, nil); err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "Database error updating admins for project '%s'", p.oldProject))
```

Comment:
The project you're actually updating is the new project, not the old project

Reply:
I had thought that at this point in the function "projectToCopy" is referencing the new project, since the newly generated Id has been set and we run p.sc.CreateProject(projectToCopy, u)?

Reply:
Or were you referring to the error message

Reply:
the error message 👍